### PR TITLE
Analysis script erv

### PR DESCRIPTION
**Pull request by dan-du-car**
# PR Details
## Description

Update script to read and parse log file and save the relevant information into excel.
Add plot script to read excel file, process the data to generate the result based on the metric requirements.
## Related Issue
https://github.com/usdot-fhwa-OPS/V2X-Hub/issues/515

## Motivation and Context
ERV freight testing

## How Has This Been Tested?
Local integration

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: scripts/v2xhub_log_analysis_erv.py [hunk 1]
```diff
@@ -25,6 +25,52 @@ def get_filenames():
...
+                        continue
+                if metric_field_title not in fields_dict.keys():
+                    fields_dict[metric_field_title] = []                
+                fields_dict[metric_field_title].append(metric_field_value)
+                fields_dict["Time (UTC)"].append(time.replace('status:200',''))            
+        file_stream.close()
+    except FileNotFoundError:
+        print(f'Log file not found {inputfile}')
+        exit()
+=======
```
Comment (paulbourelly999) [inline scripts/v2xhub_log_analysis_erv.py:73]: It looks like in several places in this PR you still have the merge conflict delimiters.

File: scripts/v2xhub_log_analysis_erv.py [hunk 2]
```diff
@@ -63,6 +109,7 @@ def process_input_log_file(inputfile, search_keyword):
                 fields_dict[metric_field_title] = []
             fields_dict[metric_field_title].append(metric_field_value)
     file_stream.close()
+>>>>>>> k900
```
Comment (paulbourelly999) [inline scripts/v2xhub_log_analysis_erv.py:112]: merge conflict delimiter

File: scripts/v2xhub_log_analysis_erv.py [hunk 3]
```diff
@@ -75,11 +122,22 @@ def process_input_log_file(inputfile, search_keyword):
 def main():    
     inputfile, outputfile = get_filenames()
     search_metric_keywords = {
+<<<<<<< HEAD
```
Comment (paulbourelly999) [inline scripts/v2xhub_log_analysis_erv.py:125]: merge conflict delimiter

File: scripts/v2xhub_log_analysis_erv.py [hunk 4]
```diff
@@ -75,11 +122,22 @@ def process_input_log_file(inputfile, search_keyword):
...
+                              'FER-4-5-8-9': 'Receive BSM',
+                              'FER-10-11-1': 'Forwarding message to cloud via curl',
+                              'FER-10-11-2': 'Successfully forwarded message to cloud via curl',
+                              'FER-TBD-1': 'Received ERV BSM from cloud',
+                              'FER-TBD-2': 'Received ERV BSM from cloud and broadcast ERV BSM delay(ms)',
+                              'FER-TBD-3': 'Incoming BSM is not from Emergency Response Vehicle (ERV)',
+                              'FER-TBD-4': 'Forward ERV BSM to cloud',
+                              'FER-TBD-5': 'Received ERV BSM and forward ERV BSM to cloud delay (ms)',
+=======
```
Comment (paulbourelly999) [inline scripts/v2xhub_log_analysis_erv.py:134]: Merge conflict delimiter

File: scripts/v2xhub_log_analysis_erv.py [hunk 5]
```diff
@@ -75,11 +122,22 @@ def process_input_log_file(inputfile, search_keyword):
...
                               'FER-5-6': 'Incoming BSM is not from Emergency Response Vehicle (ERV)',
                               'FER-9-10-11': 'Forward ERV BSM to cloud:',
                               'FER-12-1': 'Received ERV BSM and forward ERV BSM to cloud delay (ms)',
                               'FER-TBD-1': 'Received ERV BSM from cloud:',
                               'FER-TBD-2': 'Received ERV BSM from cloud and broadcast ERV BSM delay(ms)'
+>>>>>>> k900
```
Comment (paulbourelly999) [inline scripts/v2xhub_log_analysis_erv.py:140]: merge conflict delimiter

File: scripts/v2xhub_log_analysis_erv.py [hunk 6]
```diff
@@ -25,6 +25,52 @@ def get_filenames():
 
 
 def process_input_log_file(inputfile, search_keyword):
+<<<<<<< HEAD
```
Comment (paulbourelly999) [inline scripts/v2xhub_log_analysis_erv.py:28]: merge conflict delimiter

Comment (dan-du-car) [reply scripts/v2xhub_log_analysis_erv.py]: https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/524/commits/354ef421229caf43b9f18dec541cc4e3ded3384d

File: scripts/v2xhub_log_analysis_erv.py (hunk 4 above)
Comment (dan-du-car) [reply scripts/v2xhub_log_analysis_erv.py]: https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/524/commits/354ef421229caf43b9f18dec541cc4e3ded3384d

File: scripts/v2xhub_log_analysis_erv.py (hunk 3 above)
Comment (dan-du-car) [reply scripts/v2xhub_log_analysis_erv.py]: https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/524/commits/354ef421229caf43b9f18dec541cc4e3ded3384d

File: scripts/v2xhub_log_analysis_erv.py (hunk 1 above)
Comment (dan-du-car) [reply scripts/v2xhub_log_analysis_erv.py]: https://github.com/usdot-fhwa-OPS/V2X-Hub/commit/354ef421229caf43b9f18dec541cc4e3ded3384d

**Comment by JonSmet** [comment]
Closing PR as the script and associated files will be merged into the carma-analytics-fotda repository instead. Related PR: https://github.com/usdot-fhwa-stol/carma-analytics-fotda/pull/15

**Pull request closed without merging**